src/main/java/wellnus/reflection/feature/QuestionList.java

```diff
@@ -191,6 +194,21 @@ public void addFavListIndex(int indexToAdd) throws StorageException {
         UI.printOutputMessage(ADD_FAV_SUCCESS_ONE + this.questions.get(indexToAdd).toString() + ADD_FAV_SUCCESS_TWO);
     }
 
+    /**
+     * Remove the index of a liked question from the fav list.<br/>
+     * <br/>
+     * Indexes of all favorite questions will be stored in data file every time a question is removed.
+     *
+     * @param indexToRemove The index of question to be removed from fav list.
+     * @throws StorageException If data fails to be stored properly.
+     */
+    public void removeFavListIndex(int indexToRemove) throws StorageException {
+        this.dataIndex.get(INDEX_ZERO).remove(indexToRemove);
```

Comment:
Very minor code style improvement here, but it would be more readable and easier to understand if you split this line into 2 lines:
```
Set<Integer> favList = this.dataIndex.get(INDEX_ZERO);
favList.remove(indexToRemove);
```